### ISSUE TITLE
Deleted from disk

### ISSUE THREAD

**Issue by adamos2468** [humy2833/FTP-Simple#175]
When I click on unloaded directories I get a tab with the name [Dir]<directory name>(deleted from disk) and the directory disappears from the navigation panel. Additionally I get the message FTP reopen!!.

**Comment by paulbrab** [comment]
I thought the same but then realised that the directory appeared as a normal directory at the top of the list

**Comment by LuisEduardoPS** [comment]
I am having the exact same issue. Did you find a solution @adamos2468 ?

**Comment by adamos2468** [comment]
> > When I click on unloaded directories I get a tab with the name [Dir](deleted from disk) and the directory disappears from the navigation panel. Additionally I get the message FTP reopen!!.
> 
> 
> 
> I am having the exact same issue. Did you find a solution @adamos2468 ?

No, but now I use the new SSH remote development extension. Here is a tutorial: https://code.visualstudio.com/docs/remote/ssh-tutorial
